src/backend/dx12/src/lib.rs

```diff
@@ -917,6 +917,9 @@ impl hal::Instance for Instance {
                     min_buffer_copy_offset_alignment: d3d12::D3D12_TEXTURE_DATA_PLACEMENT_ALIGNMENT as _,
                     min_buffer_copy_pitch_alignment: d3d12::D3D12_TEXTURE_DATA_PITCH_ALIGNMENT as _,
                     min_uniform_buffer_offset_alignment: 256, // Required alignment for CBVs
+                    framebuffer_color_samples_count: 0b111,
```

Comment:
any particular reason for 7 here?

Reply:
Should be 0b101 (4_BIT | 1_BIT). Basically the sample counts supported for each render target.